packages/block-library/src/template-part/index.php

```diff
@@ -110,12 +110,16 @@ function render_block_core_template_part( $attributes ) {
 	$content = do_shortcode( $content );
 
 	if ( empty( $attributes['tagName'] ) ) {
-		$area_tags = array(
-			WP_TEMPLATE_PART_AREA_HEADER        => 'header',
-			WP_TEMPLATE_PART_AREA_FOOTER        => 'footer',
-			WP_TEMPLATE_PART_AREA_UNCATEGORIZED => 'div',
-		);
-		$html_tag  = null !== $area && isset( $area_tags[ $area ] ) ? $area_tags[ $area ] : $area_tags[ WP_TEMPLATE_PART_AREA_UNCATEGORIZED ];
+		$defined_areas = gutenberg_get_allowed_template_part_areas();
+		$area_tag      = 'div';
+		if ( null !== $area ) {
```

Comment:
I can't see how this could be ever `null` 🤔

Reply:
Oh good point.  I was refactoring logic that already existed, but looking closer it is initially defined with the uncategorized area now anyways.